python/rascaline/rascaline/utils/power_spectrum.py

```diff
@@ -183,9 +190,18 @@ def compute(
         ]
         assert spherical_expansion_1.keys.names == expected_key_names
         assert spherical_expansion_1.property_names == ["n"]
-        spherical_expansion_1 = spherical_expansion_1.keys_to_properties(
-            "species_neighbor"
-        )
+
+        if fill_species_neighbor:
+            keys_to_move = Labels(
+                names="species_neighbor",
+                values=np.unique(
+                    spherical_expansion_1.keys["species_neighbor"]
+                ).reshape(-1, 1),
+            )
```

Comment:
I think I'd be fine with this being the only behavior: we are not introducing too much zeros since we are moving the two spherical expansions separately, so we could just always do this and not have an extra option.